### PR TITLE
release(ad/smartbanner): removes fixed positioning in favor of static

### PR DESCRIPTION
**Pull request by davecarter**
In order to avoid unwanted interactions with the new mobile actionBar the SmartBanner positioning has to change from absolute to static.

Instead of simply remove one style positioning with the other, I'm adding a prop `static` to define the component layout.

Please review @SUI-Components/developers

### CODE REVIEW

**Review by ruben-martin-lozano — APPROVED** [review]

**Review by aleanez — APPROVED** [review]

File: components/ad/smartbanner/src/index.js [hunk 1]
```diff
@@ -17,8 +18,12 @@ export default class AdSmartbanner extends Component {
   }
 
   render () {
+    const positioning = cx('sui-AdSmartbanner', {
```
Comment (ruben-martin-lozano) [inline components/ad/smartbanner/src/index.js]: Por standard, esta variable la llamaria `className` en lugar de positioning: lo primero que me viene por positioning seria "top, bottom, etc.", pero el valor de esta variable son precisamente className/s

File: components/ad/smartbanner/package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
   "license": "ISC",
   "dependencies": {
     "@schibstedspain/sui-svgiconset": "1",
-    "@schibstedspain/suistudio-fatigue-deps": "github:SUI-Components/suistudio-fatigue-deps"
+    "@schibstedspain/suistudio-fatigue-deps": "github:SUI-Components/suistudio-fatigue-deps",
+    "classnames": "2.2.5"
```
Comment (kikoruiz) [inline components/ad/smartbanner/package.json]: `suistudio-fatigue-deps` already set `classnames` as dependency.

File: components/ad/smartbanner/src/index.js (hunk 1 above)
Comment (davecarter) [reply components/ad/smartbanner/src/index.js]: me cuesta buscar un nombre apropiado a las cosas.. piensa que mi perro se llamaba Render

File: components/ad/smartbanner/src/index.scss [hunk 3]
```diff
@@ -58,4 +58,8 @@
   &-hidden {
     margin-top: 0;
   }
+
+  &-static {
```
Comment (kikoruiz) [inline components/ad/smartbanner/src/index.scss]: It should be a modifier, or even better a simple state: `.is-static`.

Comment (davecarter) [reply components/ad/smartbanner/src/index.scss]: I used the previously name convention as in: `-hidden`

Comment (kikoruiz) [reply components/ad/smartbanner/src/index.scss]: I didn't notice it, but it should be changed as well.

File: components/ad/smartbanner/src/index.js [hunk 4]
```diff
@@ -42,7 +47,8 @@ AdSmartbanner.propTypes = {
   imageUrl: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
   text: PropTypes.string.isRequired,
-  buttonText: PropTypes.string.isRequired
+  buttonText: PropTypes.string.isRequired,
+  static: PropTypes.bool
```
Comment (danivm) [inline components/ad/smartbanner/src/index.js]: You should to define defaultProps for all non-required props

File: components/ad/smartbanner/src/index.scss [hunk 5]
```diff
@@ -51,11 +51,15 @@
...
 
-  &-visible {
+  &--visible {
     margin-top: 70px;
   }
 
-  &-hidden {
+  &--hidden {
     margin-top: 0;
   }
```
Comment (ruben-martin-lozano) [inline components/ad/smartbanner/src/index.scss]: Where are those "visible" and "hidden" classes used?

Comment (davecarter) [reply components/ad/smartbanner/src/index.scss]: anywhere 😲  !

Comment (kikoruiz) [reply components/ad/smartbanner/src/index.scss]: Well, another thing to discuss is... why are they called visible and hidden? But... "that's another war" 😄

**Review by kikoruiz — APPROVED** [review]

File: components/ad/smartbanner/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@schibstedspain/sui-ad-smartbanner",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (carlosvillu) [inline components/ad/smartbanner/package.json]: Please dont touch the version of the package. Is an automatic process

**Review by carlosvillu — CHANGES_REQUESTED** [review]

File: components/ad/smartbanner/src/index.scss (hunk 5 above)
Comment (davecarter) [reply components/ad/smartbanner/src/index.scss]: Now I know 😅 
Turns out this modifier is appended to the App Container DIV:
`<div id="App" class="sui-AdSmartbanner-visible">`

File: components/ad/smartbanner/src/index.js [hunk 7]
```diff
@@ -17,8 +18,12 @@ export default class AdSmartbanner extends Component {
   }
 
   render () {
+    const className = cx('sui-AdSmartbanner', {
+      'sui-AdSmartbanner-static': this.props.static
```
Comment (JoanClaret) [inline components/ad/smartbanner/src/index.js]: What do you think about defining the static class as a modifier? something like `is-static`.

**Review by carlosvillu — APPROVED** [review]

**Review by danivm — APPROVED** [review]
🚀

File: components/ad/smartbanner/src/index.scss (hunk 5 above)
Comment (JoanClaret) [reply components/ad/smartbanner/src/index.scss]: Do you think this is a component responsability? i mean, looks like this class should be added somewhere outside the component.

Comment (ruben-martin-lozano) [reply components/ad/smartbanner/src/index.scss]: Tal cual @JoanClaret!

File: components/ad/smartbanner/src/index.scss [hunk 8]
```diff
@@ -51,11 +51,11 @@
     }
   }
 
-  &-visible {
+  &--visible {
     margin-top: 70px;
```
Comment (JoanClaret) [inline components/ad/smartbanner/src/index.scss]: magic number? `70px`

Comment (JoanClaret) [reply components/ad/smartbanner/src/index.scss]: maybe you should use `$h-ad-smartbanner`

Comment (davecarter) [reply components/ad/smartbanner/src/index.scss]: yes! you're right. Done

**Pull request merged**